src/test/variable/VYToken.t.sol

```diff
@@ -259,75 +255,3 @@ contract VYTokenTest is VYTokenZeroState {
         assertLe(principalAmount, vyToken.convertToPrincipal(INK));
     }
 }
-
-contract FlashLoanEnabledStateTests is FlashLoanEnabledState {
```

Comment:
Why remove all these tests exactly?

Reply:
Since, flash loan was removed the tests are removed as well.